utils/post_utils.jsx

```diff
@@ -116,23 +120,15 @@ export function shouldShowDotMenu(post) {
     return false;
 }
 
-export function containsAtMention(text, key) {
-    if (!text || !key) {
+export function containsAtChannel(text) {
+    // Don't warn for slash commands
+    if (!text || text.startsWith('/')) {
         return false;
     }
 
-    // This doesn't work for at mentions containing periods or hyphens
-    return !text.startsWith('/') && new RegExp(`\\B${key}\\b`, 'i').test(removeCode(text));
-}
-
-// Returns a given text string with all Markdown code replaced with whitespace.
-export function removeCode(text) {
-    // These patterns should match the ones in app/notification.go, except JavaScript doesn't
-    // support \z for the end of the text in multiline mode, so we use $(?![\r\n])
-    const codeBlockPattern = /^[^\S\n]*[`~]{3}.*$[\s\S]+?(^[^\S\n]*[`~]{3}$|$(?![\r\n]))/mg;
-    const inlineCodePattern = /`+(?:.+?|.*?(.*?\S.*?\n)*.*?)`+/mg;
+    const mentionableText = formatWithRenderer(text, new MentionableRenderer());
 
-    return text.replace(codeBlockPattern, '').replace(inlineCodePattern, ' ');
+    return (/\B@(all|channel)\b/i).test(mentionableText);
 }
```

Comment:
Why does @here not included here.

Reply:
We didn't check for `@here` before, so I kept it the same